deploy/crds/org_v1_che_crd.yaml

```diff
@@ -231,6 +231,41 @@ spec:
                     `latest` images, and `IfNotPresent` in other cases.
                   type: string
               type: object
+            imagePuller:
+              description: Kubernetes Image Puller configuration
+              properties:
+                enable:
+                  description: Install and configure a KubernetesImagePuller. If true
+                    and no spec is provided, create a default KubernetesImagePuller.  If
+                    false, the the KubernetesImagePuller will deleted, and the operator
+                    will be uninstalled, regardless of whether or not a spec is provided.
```

Comment:
`the the KubernetesImagePuller` -> double `the` +  will `be` deleted + I would recommend removing Kubernetes wording here.

Reply:
The type of the object that is created is a `KubernetesImagePuller` custom resource.  I think that is important to point out in the autocomplete when you are configuring the che-operator, so that if a problem arises the user can check the `KubernetesImagePuller` object.